cmd/sachet/main.go

```diff
@@ -66,6 +66,11 @@ func main() {
 			return
 		}
 
+		// default is set to "text" messages
+		if receiverConf.Type == "" {
+			receiverConf.Type = "text"
+		}
```

Comment:
I don't think this is needed, it should be fine if it defaults to an empty string and then each provider should just handle the empty string as it's default (providers might not have type `text` and it might be confusing for them to receive it :) )